content/en/admin/upgrading.md

```diff
@@ -18,15 +18,30 @@ su - mastodon
 
 And navigate to the Mastodon root directory:
 
-Download the releases’s code, assuming that the version is called `v2.5.0`:
+Download the latest releases’s code:
 
 ```bash
 git fetch --tags
-git checkout v2.5.0
+git checkout $(git tag -l | grep -v 'rc[0-9]*$' | sort -V | tail -n 1)
 ```
 
 **The release page contains a changelog, and below it, upgrade instructions**. This is where you would execute them, for example, if the release mentions that you need to re-compile assets, you would execute:
 
+Update the Ruby and JavaScript dependencies
```

Comment:
Please mind the above paragraph:

> **The release page contains a changelog, and below it, upgrade instructions**. This is where you would execute them, for example, if the release mentions that you need to re-compile assets, you would execute:

Reply:
Generally speaking while the set of instructions is *usually* the same from release-to-release, *sometimes* it is not the case, so I don't want to give sysadmins false general instructions here.

Reply:
The release v3.1.1 hasn't any upgrade notes but `bundle install` is needed due to Gemfile changes.
What if a sysadmin skip some releases? He needs to cumulate the mentions in the release notes?
What if a sysadmin is lazy to read the release notes?
Isn't it better to run db:migrate w/o changes than missing a db migration?
Just my two cents :)